src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas2/SubmissionsController.kt

```diff
@@ -114,7 +114,7 @@ class SubmissionsController(
 
   override fun submissionsApplicationIdStatusUpdatesPost(
     applicationId: UUID,
-    statusUpdate: Cas2ApplicationStatusUpdate,
```

Comment:
I feel like I want to keep 'Assessment' in there somehow to make it super clear - and take the emphasis off 'status' because I don't think these are technically statuses (as in they don't take anything through a state machine, they are more like records in time of an event) and they can only be created currently by an Assessor...